yarn.lock

```diff
@@ -6089,7 +6094,7 @@ source-map-support@^0.5.6, source-map-support@~0.5.19:
     buffer-from "^1.0.0"
     source-map "^0.6.0"
 
-source-map-url@^0.4.0, source-map-url@^0.4.1:
+source-map-url@^0.4.0:
```

Comment:
now that 0.4.1 is removed, what's still bringing this in to the dep graph? :thinking:

Reply:
it is some deep dependency (source-map-resolve < snapdragon <  nanomatch ....) 😬